src/pages/overview/document-generation-api/quickstarts/python/index.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Python | Quickstarts | Document Generation API | Adobe PDF Services
+---
+
+# Quickstart for Adobe Document Generation API (Python)
+
+To get started using Adobe Document Generation API, let's walk through a simple scenario - using a Word document as a template for dynamic receipt generation in PDF. In this guide, we will walk you through the complete process for creating a program that will accomplish this task.
+
+## Prerequisites
+
+To complete this guide, you will need:
+
+* [Python](https://www.python.org/downloads/) - Python 3.10 or higher is required.
+* An Adobe ID. If you do not have one, the credential setup will walk you through creating one.
+* A way to edit code. No specific editor is required for this guide.
+
+
+## Step One: Getting credentials
+
+1) To begin, open your browser to <https://acrobatservices.adobe.com/dc-integration-creation-app-cdn/main.html?api=pdf-services-api>. If you are not already logged in to Adobe.com, you will need to sign in or create a new user. Using a personal email account is recommend and not a federated ID.
+
+![Sign in](./shot1.png)
+
+2) After registering or logging in, you will then be asked to name your new credentials. Use the name, "New Project".
+
+3) Change the "Choose language" setting to "Python".
+
+4) Also note the checkbox by, "Create personalized code sample." This will include a large set of samples along with your credentials. These can be helpful for learning more later.
+
+5) Click the checkbox saying you agree to the developer terms and then click "Create credentials."
+
+![Project setup](./shot2_spc.png)
+
+6) After your credentials are created, they are automatically downloaded:
+
+![Credentials](./shot3_spc.png)
+
+## Step Two: Setting up the project
+
+1) In your Downloads folder, find the ZIP file with your credentials: PDFServicesSDK-Python (Extract, Auto-tag)Samples.zip. If you unzip that archive, you will find a folder of samples and the `pdfservices-api-credentials.json` file.
+
+![Samples](./shot5_spc.png)
+
+2) Take the `pdfservices-api-credentials.json` file and place it in a new directory.
+
+3) At the command line, change to the directory you created, and run the following command to install the Python SDK: `pip install pdfservices-sdk`.
+
+![alt](shot7.png)
```

Comment:
Please update the screenshot, below screenshot show old python version confusing users

<img width="794" alt="image" src="https://github.com/AdobeDocs/pdfservices-api-documentation/assets/119481807/1342c473-76a0-405d-8f92-62779f337821">

Reply:
Applicable in all 4 locations of quickstarts